src/components/pages/Users/Users.js

```diff
@@ -421,17 +445,57 @@ const Users = () => {
         sortType: reactTableNaturalSort,
       },
       {
-        Header: 'Admin',
+        Header: () => (
+          <>
+            <LabelContainer>
+              <div>Admin</div>
+              {isHelperTextShowing && currentHelperTextLabel === 'admin' ? (
+                <ColumnHeaderToolTip helperText={language.tooltipText.admin} left="-4em" />
+              ) : null}
+              <IconButton type="button" onClick={(event) => handleInfoIconClick(event, 'admin')}>
+                <IconInfo aria-label="info" />
+              </IconButton>
+            </LabelContainer>
+          </>
+        ),
         accessor: 'admin',
         disableSortBy: true,
       },
       {
-        Header: 'Collector',
+        Header: () => (
+          <>
+            <LabelContainer>
+              <div>Collector</div>
+              {isHelperTextShowing && currentHelperTextLabel === 'collector' ? (
+                <ColumnHeaderToolTip helperText={language.tooltipText.collector} left="-2em" />
+              ) : null}
+              <IconButton
+                type="button"
+                onClick={(event) => handleInfoIconClick(event, 'collector')}
+              >
+                <IconInfo aria-label="info" />
+              </IconButton>
+            </LabelContainer>
+          </>
+        ),
+
         accessor: 'collector',
         disableSortBy: true,
       },
       {
-        Header: 'Read-Only',
+        Header: () => (
+          <>
+            <LabelContainer>
+              <div>Read-Only</div>
+              {isHelperTextShowing && currentHelperTextLabel === 'readOnly' ? (
+                <ColumnHeaderToolTip helperText={language.tooltipText.readOnly} left="-1em" />
```

Comment:
feels a bit hacky passing custom css in here, hoping to resolve swapping polygon clip path percentages to more precise amounts